### DISCUSSION TITLE
Welcome to intel-nuc10 Discussions!

### DISCUSSION THREAD

**Discussion by mrasong** [hackintosh-club/intel-nuc10#62] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by kclgh** [comment]
hello mrasong,
Happy holiday! Thank you for all the great work which make it so much easier to install hackintosh so quickly and seamless on my Nuc10. It boot up in first run and was amazing! Only issue I found later is when I have both the thunderbolt3 usb-c and hdmi connected to 2 monitors (an LG and a Dell), it would boot up to one black screen and one green screen and failing provide the normal login screen. However when I boot up with only one monitor connected (doesn't matter usb-c or hdmi), I got to login as normal and after that if I connect the second monitor it work just fines. Any pointers you may kindly provide to fix this? Thanks much!

**Reply by mrasong** [reply]
That's what you found it.
Two ways you can do.
1. Connect only one monitor when system start up, then plug the other one after the login window. 
2. Never shut down your NUC ;)

**Reply by kclgh** [reply]
Thanks. I was hoping to be able to boot with 2 like I did on my desktop PC which is also a Z490 with TB3 and HDMI out.

**Comment by berg-maia** [comment]
Its an old topic but maybe help someone.

I installed a few days ago on my NUC10i3FHN

I had to remove
- IOSkywalkFamily.kext (Kernel Panic)
- IntelBluetoothInjector.kext (No needed and only works without it)

Thanks for the EFI
